packages/astro/src/core/preview/index.ts

```diff
@@ -33,7 +33,7 @@ export default async function preview(
 		throw new Error(`[preview] No adapter found.`);
 	}
 	if (!settings.adapter.previewEntrypoint) {
-		throw new Error(`[preview] adapter does not have previewEntrypoint.`);
+		throw new Error(`[preview] This adapter does not support the preview command.`);
```

Comment:
Is there enough context here for this to be clear what “this adapter” is?

If I’m reading the source correctly and the `settings.adapter` is equivalent to [this object](https://github.com/withastro/astro/blob/b77200f42399ea31b0045bc0e6bfe2c1c5ccc970/packages/integrations/node/src/index.ts#L5-L11), maybe we can even do this?

```suggestion
		throw new Error(`[preview] The ${settings.adapter.name} adapter does not support the preview command.`);
```

With the idea being an error log like:

```
[preview] The @astrojs/node adapter does not support the preview command.
```

Reply:
Oh yes, this is much better, thank you!